### PR TITLE
Allow user to control logging level

### PR DESCRIPTION
**Pull request by AleksanderWWW**
## Before submitting checklist

- [x] Did you **update the CHANGELOG**? (not for test updates, internal changes/refactors or CI/CD setup)
- [ ] Did you **ask the docs owner** to review all the user-facing changes?

### CODE REVIEW

File: src/neptune/internal/utils/logger.py [hunk 1]
```diff
@@ -49,12 +49,11 @@ def stream(self):
         return sys.stdout
 
 
-def get_logger(with_prefix: bool = True, loglevel: int = logging.DEBUG):
+def get_logger(with_prefix: bool = True):
```
Comment (Raalsky) [inline src/neptune/internal/utils/logger.py:52]: Does `_handle_fork_in_child` will still going to work as previously?

File: src/neptune/internal/utils/logger.py [hunk 2]
```diff
@@ -49,12 +49,11 @@ def stream(self):
...
     if with_prefix:
         logger = logging.getLogger(NEPTUNE_LOGGER_NAME)
     else:
         logger = logging.getLogger(NEPTUNE_NO_PREFIX_LOGGER_NAME)
-    logger.setLevel(loglevel)
```
Comment (Raalsky) [inline src/neptune/internal/utils/logger.py:57]: What level are we going to have now? Maybe this should be just a part of `_set_up_logging` and `_set_up_no_prefix_logging`?

File: src/neptune/internal/utils/logger.py (hunk 1 above)
Comment (AleksanderWWW) [reply src/neptune/internal/utils/logger.py:52]: Didn't notice that - seems that our unit tests don't cover it. That might be tricky

File: src/neptune/internal/utils/logger.py (hunk 2 above)
Comment (Raalsky) [reply src/neptune/internal/utils/logger.py:57]: Debug it's for rather developers. `INFO` should be ok.

**Comment by AleksanderWWW** [comment]
## E2E
https://github.com/neptune-ai/neptune-client/actions/runs/8204730825

**Review by Raalsky — APPROVED** [review]

File: src/neptune/internal/utils/logger.py (hunk 1 above)
Comment (szysad) [reply src/neptune/internal/utils/logger.py]: @Raalsky good observation!

**Pull request merged**